### PR TITLE
add voice recognize library / add minimal ui for testing recognize feature

### PR DESCRIPTION
**Pull request by igor90007**

### CODE REVIEW

File: src/containers/General/Main.tsx [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+    try {
+      await Voice.start('en-US')
+    } catch (e) {
+      console.log(e, 'onStartButtonPressErr')
+    }
+  }
+
+  private async onStopPress() {
+    try {
+      await Voice.stop()
```
Comment (ddzirt) [inline src/containers/General/Main.tsx]: Cool, but how does it help us to even test this feature? Is it working? Output the result somewhere at least. And make it relevant to what you are doing within this project.

File: src/containers/General/Main.tsx [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+    } catch (e) {
+      console.log(e, 'onStopPressErr')
+    }
   }
 
-  private testFunc = () => {
-    //
+  private onSpeechResultsHandler = (event: object) => {
+    console.log(event, 'onSpeechResultsHandler')
```
Comment (ddzirt) [inline src/containers/General/Main.tsx]: This is not enough. Make use of this in the app.

File: src/containers/General/Main.tsx (hunk 1 above)
Comment (igor90007) [reply src/containers/General/Main.tsx]: I've just updated for testing

File: src/containers/General/Main.tsx (hunk 2 above)
Comment (igor90007) [reply src/containers/General/Main.tsx]: I've added minimal functionality for testing

File: src/containers/General/Main.tsx (hunk 1 above)
Comment (ddzirt) [reply src/containers/General/Main.tsx]: ?

**Pull request merged**